docs/operator-manual/user-management/index.md

```diff
@@ -446,6 +446,59 @@ Add a `rootCA` to your `oidc.config` which contains the PEM encoded root certifi
       -----END CERTIFICATE-----
 ```
 
+## External JWT Authentication
+
+Argo CD can be configured to verify JSON Web Tokens (JWTs) issued by an external authentication provider. This allows you to integrate Argo CD with existing authentication systems that issue JWTs.
+
+To configure external JWT authentication, add the JWT configuration to the `argocd-cm` ConfigMap:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: argocd-cm
+  namespace: argocd
+  labels:
+    app.kubernetes.io/name: argocd-cm
+    app.kubernetes.io/part-of: argocd
```

Comment:
is this actually needed? wouldn't it be added by argocd if the object was created by argocd?

Reply:
I'm not certain, but it was kept in order to stick with the pattern seen further down in the doc:
https://argo-cd.readthedocs.io/en/stable/operator-manual/user-management/#sso-further-reading

Reply:
Fair. At some point I should probably talk to the owners about this and probably remove most of them.

My understanding is that those labels tend to be added by ArgoCD itself when it creates objects which means it shouldn't be included in someone's declarative content.